main/export-list.js

```diff
@@ -193,6 +195,19 @@ class ExportList {
       openEditorWindow(exp.inputPath, exp.originalFps);
     }
   }
+
+  dragExport(createdAt) {
+    track('export/history/drag/recording');
+    const exportsWindow = getExportsWindow();
+
+    const exp = this.exports.find(exp => exp.createdAt === createdAt);
```

Comment:
`exp` => `export`

Reply:
This is copy-pasted from above, should i change that too? or make a function to get an export by createdAt?

Reply:
> or make a function to get an export by createdAt?

👍

Reply:
`export`  seems to be restricted keyword?
![image](https://user-images.githubusercontent.com/5027156/51110526-4e550100-17f9-11e9-8163-eafe72b31a9a.png)
![image](https://user-images.githubusercontent.com/5027156/51110530-514ff180-17f9-11e9-8398-487265302ae9.png)

Reply:
I remember that haha. Can't use export as a variable name because it's reserved

Reply:
Ugh, I hate that. In Swift, you can at least wrap the variable name in backticks to allow using reserved names.

Reply:
`exp` => `exportItem` then?

Reply:
yep, that's better